lib/applications/schema.ts

```diff
@@ -261,7 +264,7 @@ export default `
     usesAccessibleConvertedVan: Boolean!
     requiresWiderParkingSpace: Boolean!
     rcdUserId: Int
-
+    receiveEmailUpdates: Boolean
```

Comment:
Right now this will break the non-admin renewal flow since theres no option for specifying this field yet. I'll still make this change but it will be hard coded to `false` (the default value) for now to be addressed later, similar to how `usesAccessibleConvertedVan` and `requiresWiderParkingSpace` are currently hard coded.

Reply:
That's a good workaround, thanks.